src/components/Errors.jsx

```diff
@@ -1,6 +1,10 @@
 const Errors = (error) => {
-    const errorCode = error.error.error.status
-    const errorMessage = error.error.error.data.msg
-    return <p>Error: {errorCode} - {errorMessage}</p>
+    if(error.error.msg) {
+return (<p>{error.error.msg}</p>)
```

Comment:
I'd think about whether you want to show EVERYTHING in your error to your user - only in certain circumstances do you want to actually send the message. More often than not the code will be enough with an 'oops, something went wrong!' message